cmd/dptp-controller-manager/main.go

```diff
@@ -266,6 +276,41 @@ func main() {
 		}
 	}
 
+	allManagers := map[string]controllerruntime.Manager{
```

Comment:
We created a duplicate manager for `app.ci` before, which meant that we potentially had duplicate caches. I've added this map to avoid that in the future.